apps/web/src/services/asset-inventory/collector/create-collector/modules/AttachedServiceAccountForm.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+    <div class="attached-service-account-form">
+        <!--        TODO: translation-->
+        <p-field-group :label="props.title || $t('Attached Service Account')"
+                       :invalid="invalidState.selectedAttachedServiceAccount"
+                       :invalid-text="invalidTexts.selectedAttachedServiceAccount"
+                       :valid="state.isAttachedServiceAccountValid"
+                       :valid-text="$t('Good')"
+                       :required="true"
+        >
+            <p-radio-group class="attached-service-account-radio-group">
+                <p-radio v-for="(item) in attachedServiceAccountList"
+                         :key="`${item.name}`"
+                         :value="item.name"
+                         :selected="state.selectedAttachedServiceAccountType"
+                         @change="handleChangeAttachedServiceAccountType"
+                >
+                    {{ item.label }}
+                </p-radio>
+            </p-radio-group>
+            <p-select-dropdown class="attached-service-account-dropdown"
+                               :selected="state.selectedAttachedServiceAccountType"
+                               :items="attachedServiceAccountList"
+                               @update:selected="handleChangeAttachedServiceAccountType"
+            />
```

Comment:
nitpick: Can you add comments that they are shown optionally by device width? :)